src/TritonRoute/src/dr/FlexDR_maze.cpp

```diff
@@ -1809,7 +1809,9 @@ void FlexDRWorker::route_queue(FlexGCWorker& gcWorker)
   gcWorker.resetTargetNet();
   gcWorker.setEnableSurgicalFix(true);
   gcWorker.main();
-
+   if (graphics_) {
+    graphics_->show();
```

Comment:
We start startWorker() so this would seem to be endWorker().  I'm not sure why you put it ahead of storing the markers & patch wires.

Reply:
The idea is to have a function to show routing content regardless of the moment/context it is called. startWorker() makes sense because it does some initialization in FlexDR_graphics, but endWorker() doesnt IMO because it is only showing routing content. 
I placed in this point because I needed to see what was going on in this point of the code, but I can move it to the place you spoke of.